### PR TITLE
Post testing fixes

### PR DESCRIPTION
**Pull request by ro-tex**
# PULL REQUEST

## Overview

These are minor adjustments that I came up with while testing.

TL;DR:
 - when we get an non-breaking error (i.e. we continue processing other skylinks) while processing a skylink, we want to wait for a bit before moving on. this is both to avoid a tight loop around a faulty skylink as well giving more time for the cause of the error to go away, e.g. skyd to finish loading
 - slightly better logging

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [ ] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

### CODE REVIEW

File: workers/scanner.go [hunk 1]
```diff
@@ -171,12 +171,30 @@ func (s *Scanner) pinUnderpinnedSkylinks() {
...
-		// Block until the pinned skylink becomes healthy or until a timeout.
-		s.waitUntilHealthy(skylink, sp)
+		// We only check the error if we want to continue scanning. The error is
+		// already logged and the only indication it gives us is whether we
+		// should wait for the file we pinned to become healthy or not. If there
+		// is an error, then there is nothing to wait for.
+		if err == nil {
+			// Block until the pinned skylink becomes healthy or until a timeout.
+			s.waitUntilHealthy(skylink, sp)
+		} else {
```
Comment (MSevey) [inline workers/scanner.go:185]: nit: I would put a `continue` after the `s.waitUntilHealthy` and then remove the else to have the select de-indented.

**Review by MSevey — DISMISSED** [review]

File: workers/scanner.go (hunk 1 above)
Comment (ro-tex) [reply workers/scanner.go]: Might as well handle it right away. I used to opportunity to fix some typos and such as well.

**Review by MSevey — APPROVED** [review]

**Review by m-cat — APPROVED** [review]
